test_binheap.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+from binheap import Binheap
+
+
+# Create an empty heap
+def test_create_empty():
+    bh = Binheap()
+    assert isinstance(bh, Binheap)
+
+# Push onto a heap
+def test_push():
```

Comment:
presumably each of the asserts below tests a separate condition.  Write separate tests.  Here, if the first assertion fails, you have no idea if the rest would have passed.  So a single failure can mask other problems.